tests/ballerina-integration-test/src/test/java/org/ballerinalang/test/service/websocket/WebSocketInitServer.java

```diff
@@ -30,7 +30,7 @@
 /**
  * Facilitate the common functionality of WebSocket integration tests.
  */
-public class WebSocketIntegrationTest {
+public class WebSocketInitServer {
```

Comment:
Why have we renamed this to InitServer? We should come up with a better name here.